express/services/players.js

```diff
@@ -1,14 +1,10 @@
 const models = require('../../sequelize/models');
+const serviceUtils = require('../../utils/sequelize/serviceUtils');
 
-function getAll({ filterField, filterValue, sortField = 'id', sortDirection = 'ASC' }) {
-    const findAllOptions = {
-        ...filterField && filterValue && { where: { [filterField]: filterValue } },
-        order: [
-            [sortField, sortDirection]
-        ]
-    }
+function getAll({ filterField, filterValue, sortField, sortDirection, page, limit }) {
 
-    return models.player.findAll(findAllOptions);
+    return models.player.findAndCountAll(
+        serviceUtils.getSearchOptions(filterField, filterValue, sortField, sortDirection, page, limit));
```

Comment:
I suggest keeping the same pattern for getSearchOptions method and pass named args in the object here, so you can do it like this 
```
getAll(params) {
    serviceUtils.getSearchOption(params)
```